docs/source/exporting.rst

```diff
@@ -45,6 +45,11 @@ Converting linked SVG to PDF
 .. autoclass:: SVG2PDFPreprocessor
 
 
+Embedding images in notebooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: EmbedPreprocessor
```

Comment:
I think this should be `EmbedImagesPreprocessor`, no?

Reply:
Would say so too, its more explicit :-)

Reply:
> its more explicit

but more importantly, at the moment I don't think `EmbedPreprocessor` is an extant class, so I don't think it'll produce anything at all...